test/kpod_pause.bats

```diff
@@ -13,166 +11,97 @@ function teardown() {
 }
 
 @test "pause a bogus container" {
-    run ${KPOD_BINARY} ${KPOD_OPTIONS} pause foobar
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} pause foobar"
     echo "$output"
     [ "$status" -eq 1 ]
 }
 
 @test "unpause a bogus container" {
-    run ${KPOD_BINARY} ${KPOD_OPTIONS} unpause foobar
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} unpause foobar"
     echo "$output"
     [ "$status" -eq 1 ]
 }
 
 @test "pause a created container by id" {
-    skip "Test needs to be converted to kpod run"
-    start_crio
-    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
-    echo "$output"
-    [ "$status" -eq 0 ]
-    pod_id="$output"
-    run crioctl image pull "$IMAGE"
-    [ "$status" -eq 0 ]
-    run crioctl ctr create --config "$TESTDATA"/container_config.json --pod "$pod_id"
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} create $BB ls"
     echo "$output"
     [ "$status" -eq 0 ]
     ctr_id="$output"
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} pause "$ctr_id"
-    echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} unpause "$ctr_id"
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} pause $ctr_id"
     echo "$output"
-    [ "$status" -eq 0 ]
-    run bash -c ${KPOD_BINARY} ${KPOD_OPTIONS} ps -a --filter id="$ctr_id"
+    [ "$status" -eq 1 ]
```

Comment:
It looks like pausing created but not running containers is valid, so we should assume this should still return 0

Reply:
you lost me. pausing a container that is not running should fail right? you should only be able to pause a running container yes?

Reply:
It is valid to do an Exec() on a created but not running container, which means something is actually running in the container that can be paused. This is wierd, but valid.

Reply:
okay, so created and running states can be paused, others cannot? Will fix that in the code

Reply:
hmm, looks like that container state is `ContainerStateConfigured` when using the kpod create command. When is the state actually `ContainerStateCreated`?

Reply:
I think only kpod run does it right now (kpod start would have to also). We moved creating the container in runc out of kpod create at the request of @rhatdan  - If it's breaking compatability with Docker, we can revisit that decision.